src/main/java/strman/Strman.java

```diff
@@ -1334,5 +1336,40 @@ private static long countSubstr(String value, String subStr, boolean allowOverla
         return countSubstr(value.substring(offset), subStr, allowOverlapping, ++count);
     }
 
+    /**
+     * Aggregates the contents of n strings into a single list of tuples.
+     *
+     * @param inputs A list of strings.
+     * @return A list of strings if there are enough elements in the input and none of them is null or empty.
+     * An empty list otherwise.
+     */
+    public static List<String> zip(String... inputs) {
+        if (inputs.length < 2) {
+            return emptyList();
```

Comment:
If input.length is equal to 1 we should return the first string in list form as emptylist will confuse user. Return empty when inputs.length is 0